demisto_sdk/commands/common/errors.py

```diff
@@ -421,6 +422,15 @@ def no_docker_tag(docker_image):
     def docker_not_formatted_correctly(docker_image):
         return f'The docker image: {docker_image} is not of format - demisto/image_name:X.X'
 
+    @staticmethod
+    @error_code_decorator
+    def docker_not_on_the_latest_tag(docker_image_tag, docker_image_latest_tag, docker_image_name):
+        return f'The docker image tag is not the latest, please update it.\n' \
```

Comment:
please specify it should be the numeric value and not the tag latest

Reply:
added, though the lines below tell what should be the changed tag.